packages/webdriverio/src/commands/browser/waitUntil.ts

```diff
@@ -59,11 +59,11 @@ export default function waitUntil(
      * ensure that timeout and interval are set properly
      */
     if (typeof timeout !== 'number') {
-        timeout = this.options.waitforTimeout
+        timeout = this.options.waitforTimeout as number
     }
 
     if (typeof interval !== 'number') {
-        interval = this.options.waitforInterval
+        interval = this.options.waitforInterval as number
```

Comment:
Do we really need to cast them?

Reply:
I couldn't think of a better approach here so far... we have a few other options:
1 - try to type guard against `undefined` before passing these args to Timer (this is what broke tests before).
2 - Timer would accept `undefined` for `interval` and `timeout` (doesn't feel right unless Timer would have defaults for them)
3 - WebdriverIO.Browser.options would declare `waitforTimeout` and `waitforInterval` as mandatory (also doesn't sound like a good option 🤔 )
```
/**
  * Default timeout for all `waitFor*` commands. (Note the lowercase f in the option name.)
  * This timeout only affects commands starting with `waitFor*` and their default wait time.
  */
waitforTimeout?: number;
```

Reply:
At some point the testrunner will apply default values to it (I think at the point where we validate the options). I wasn't aware that these can be also undefined. Therefor I would argue that we could leave this as is.

Reply:
> I would argue that we could leave this as is.

Sorry, you mean with the cast?